internal/template-validator/validator/app.go

```diff
@@ -15,6 +16,7 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	kubevirtv1 "kubevirt.io/api/core/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
```

Comment:
Template validator is not using controller-runtime otherwise. I'd prefer to stick to the solution you had before. :)